webapp-django/crashstats/signature/views.py

```diff
@@ -127,16 +135,27 @@ def signature_reports(request, params):
 
     context['params'] = current_query.copy()
 
+    if '_sort' in context['params']:
+        del context['params']['_sort']
+
     if '_columns' in context['params']:
         del context['params']['_columns']
 
+    context['sort'] = request.GET.getlist('_sort') or DEFAULT_SORT
     context['columns'] = request.GET.getlist('_columns') or DEFAULT_COLUMNS
 
-    # Make sure only allowed fields are used
+    # Make sure only allowed fields are used.
+    context['sort'] = [
+        x for x in context['sort']
+        if x in allowed_fields or
+        (x.startswith('-') and x[1:] in allowed_fields)
+    ]
```

Comment:
It's fine if there is not sorting done. That's what happens at the moment anyway.